apps/dapp/src/components/Pages/Core/NewUI/RAMOSMetrics.tsx

```diff
@@ -0,0 +1,163 @@
+import type { FC } from 'react';
+import type { RAMOSMetric } from 'hooks/core/types';
+import type { ChartSupportedTimeInterval } from 'utils/time-intervals';
+
+import styled, { useTheme } from 'styled-components';
+import { format, differenceInDays } from 'date-fns';
+import { BiAxialAreaChart } from 'components/Charts';
+import Loader from 'components/Loader/Loader';
+import { useRAMOSMetrics } from 'hooks/core/subgraph';
+import { formatTimestampedChartData } from 'utils/charts';
+import { formatNumberAbbreviated, formatNumberWithCommas } from 'utils/formatter';
+import * as breakpoints from 'styles/breakpoints';
+
+type XAxisTickFormatter = (timestamp: number) => string;
+
+const RAMOS_LAUNCH_DATE = new Date(1671058907000);
+
+const tooltipLabelFormatters: Record<ChartSupportedTimeInterval, XAxisTickFormatter> = {
+  '1D': (timestamp) =>
+    `Day ${differenceInDays(timestamp, RAMOS_LAUNCH_DATE).toString()}, ${format(timestamp, 'h aaa')}`,
+  '1W': (timestamp) => `Day ${differenceInDays(timestamp, RAMOS_LAUNCH_DATE).toString()}`,
+  '1M': (timestamp) => `Day ${differenceInDays(timestamp, RAMOS_LAUNCH_DATE).toString()}`,
+  '1Y': (timestamp) => `Day ${differenceInDays(timestamp, RAMOS_LAUNCH_DATE).toString()}`,
+};
+
+const tooltipValuesFormatter = (value: number, name: string) => [formatNumberWithCommas(value), name];
+
+const tooltipValueNames = {
+  templeBurned: 'TEMPLE burned',
+  totalProfitUSD: 'Value accrual to holders (USD)',
+};
+
+const xTickFormatter = (value: number, _index: number) =>
+  `  ${differenceInDays(value, RAMOS_LAUNCH_DATE).toString()}  `;
+
+const CHART_INTERVAL: ChartSupportedTimeInterval = '1Y';
+
+export const RAMOSMetrics: FC = () => {
+  const { dailyMetrics, hourlyMetrics, isLoading, errors } = useRAMOSMetrics();
+  const theme = useTheme();
+
+  if (errors.some(Boolean)) {
+    console.error('Error fetching data', errors);
+    return null;
+  }
+
+  if (isLoading) {
+    return <Loader iconSize={48} />;
+  }
+
+  if (dailyMetrics === null || hourlyMetrics === null) {
+    console.error('Invalid subgraph response', dailyMetrics, hourlyMetrics);
+    return null;
+  }
+
+  if (dailyMetrics.data === undefined || hourlyMetrics.data === undefined) {
+    console.error('Empty subgraph response', dailyMetrics, hourlyMetrics);
+    return null;
+  }
+
+  const formattedData = formatTimestampedChartData(
+    dailyMetrics.data.metricDailySnapshots,
+    hourlyMetrics.data.metricHourlySnapshots,
+    formatDataPoint
+  );
+
+  if (formattedData === null) {
+    return <div>Empty payload</div>;
```

Comment:
I guess this would appear on the home page if there was some error formatting the data. Should we just return `null` instead?

Reply:
Yes, missed this one. Meant to make it return `null` and log the error like the other previous instances of errors. Thanks 🙏